package/config.yml

```diff
@@ -11,7 +11,7 @@ include:
   - platform.txt
   - programmers.txt
   - README.md
-index-name: package_infineon_index
+index-name: package_xmc_index
```

Comment:
This means the manifest url needs to be changed as well -> 
https://github.com/Infineon/XMC-for-Arduino/blob/4.0.0-metafiles-naming-conventions-boardname/docs/installation-instructions.rst
![image](https://github.com/user-attachments/assets/5b10fc28-7698-47a8-b9e9-b0acb69b1e88)